apps/site/assets/ts/shuttles/components/StationDropdown.tsx

```diff
@@ -15,7 +15,9 @@ const StationDropdown = ({
   const [selectedStationId, setSelectedStationId] = useState(stations[0].id);
   useEffect(
     () => {
-      onSelect(selectedStationId);
+      if (selectedStationId) {
```

Comment:
Shouldn't there always be a selected ID? Is this left over from when we had  the "Select a station" entry?